refresh/templates/extensions/WatsonConversationExtension.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+import ConversationV1
+import CloudFoundryConfig
+
+extension Conversation {
+
+   public convenience init(service: WatsonConversationService) {
+
+        let version = "<%- service.version %>"
```

Comment:
This issue, I would guess: https://github.com/IBM-Swift/generator-swiftserver/issues/207